src/commonTest/kotlin/com/github/michaelbull/result/coroutines/SuspendableBindingTest.kt

```diff
@@ -81,7 +81,7 @@ class SuspendableBindingTest {
 
         suspend fun provideZ(): Result<Int, BindingError> {
             delay(1)
-            return Ok(2)
+            return Err(BindingError)
```

Comment:
while this isnt an issue here in the non async version, thought it made more sense to do something similar here given the name of the test implies a possible second failed binding.